pkg/kubectl/cmd/cmd.go

```diff
@@ -364,7 +365,22 @@ func (h *DefaultPluginHandler) Lookup(filename string) (string, bool) {
 
 // Execute implements PluginHandler
 func (h *DefaultPluginHandler) Execute(executablePath string, cmdArgs, environment []string) error {
-	return syscall.Exec(executablePath, cmdArgs, environment)
+
+	// Windows does not support exec syscall.
+	if runtime.GOOS == "windows" {
+		cmd := exec.Command(executablePath, cmdArgs...)
+		cmd.Stdout = os.Stdout
+		cmd.Stderr = os.Stderr
+		cmd.Stdin = os.Stdin
+		cmd.Env = environment
+		err := cmd.Run()
+		if err == nil {
+			os.Exit(0)
+		}
+		return err
+	}
+
+	return syscall.Exec(executablePath, append([]string{executablePath}, cmdArgs...), environment)
```

Comment:
isn't this a behavior change to the args passed to the invoked command (prepended with executable path or not)?

Reply:
nevermind, found the call site was updated